src/mahoji/lib/abstracted_commands/minionKill.ts

```diff
@@ -463,6 +463,9 @@ export async function minionKillCommand(
 				return e.message;
 			}
 		}
+	} else {
+		boosts.push('4% for no food');
+		duration = reduceNumByPercent(duration, 4);
 	}
```

Comment:
pointless, duplicated `else`. use the above one thats already there.

Reply:
can't use the above one since nested.